packages/base-map/src/styled.ts

```diff
@@ -73,15 +73,11 @@ export const LayerSelector = styled.aside`
   }
 
   .layers-list {
-    background: rgba(255, 255, 255, 0.95);
```

Comment:
Why was this removed? I understand matching the zoom buttons but those are tighter and also have a box shadow

Reply:
I am such a non-believer.

Reply:
How about we keep the previous box shadow on hover?

Reply:
That's an acceptable compromise to me!

Reply:
I added the on-hover box shadow in https://github.com/opentripplanner/otp-ui/pull/578/commits/7f0e274f8cc99694d04136d1e934cb5fa189c51b.

Reply:
Can confirm the box shadow looks good on hover!